src/less/section-notice/base/section-notice.less

```diff
@@ -4,31 +4,21 @@
     .customBackgroundColorProperty(section-notice-default-background-color);
     .customBorderRadiusProperty(section-notice-border-radius);
 
-    flex-wrap: wrap;
+    display: flex;
     font-size: @font-size-14;
+    justify-content: flex-start;
     margin: 8px 0;
     padding: 16px;
 }
 
-div[role="region"].section-notice,
-section.section-notice {
-    display: flex;
-}
-
-span[role="region"].section-notice {
-    display: inline-flex;
-}
-
-.section-notice--attention {
-    .customBackgroundColorProperty(section-notice-attention-background-color);
-}
-
-.section-notice--confirmation {
-    .customBackgroundColorProperty(section-notice-confirmation-background-color);
+.section-notice .fake-link {
+    color: black;
+    font-weight: bold;
 }
 
-.section-notice--information {
-    .customBackgroundColorProperty(section-notice-information-background-color);
+.section-notice a {
+    color: black;
```

Comment:
Try and use a variable here. In fact, it should use our custom property mixin in order to be compatible with dark mode.

Reply:
Same comment applies to other places that you are using `color: black`